hangman/game.py

```diff
@@ -1,25 +1,79 @@
 from .exceptions import *
-
+import random
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
 
 def _get_random_word(list_of_words):
-    pass
+    #1. pick an integer from the len(list_of_words)
+    #2. return word in the list at the position of the integer
+    number_of_words = len(list_of_words)
+    if number_of_words == 0:
+        raise InvalidListOfWordsException
+    random_int = random.randint(0,number_of_words)
+    return list_of_words[random_int-1]
 
+def _mask_word(answer_word):
+    #1. count length of word
+    #2. for loop of the length of word to add one asterisk until length of maskedWord = length of word
+    if (answer_word == None or answer_word == ''):
+        raise InvalidWordException
 
-def _mask_word(word):
-    pass
+    word_length = len(answer_word)
+    masked_word = ""
+    for i in range(word_length):
+        masked_word = masked_word + "*"
+    print(masked_word)
+    return masked_word
```

Comment:
you can combine line 22 - 27 with this:-
`return len(word) * "*"`  <- this return asterisks of the same length of word